SQL/0000-00-02-Menus.sql

```diff
@@ -35,9 +35,10 @@ INSERT INTO LorisMenu (Label, OrderNumber) VALUES
      ('Candidate', 1),
      ('Clinical', 2),
      ('Imaging', 3),
-     ('Reports', 4),
-     ('Tools', 5),
-     ('Admin', 6);
+     ('Electrophysiology', 4)
```

Comment:
missing a comma here

Reply:
ooops... Should be fixed in the next commit.